packages/forms/src/elements/search/search.tsx

```diff
@@ -24,21 +25,22 @@ type PanelVisibility = 'visible' | 'hidden' | 'complete';
 
 const Search: React.FC<SearchProps> = React.memo(
 	({
-		id,
+		assistiveHint = 'When autocomplete results are available use up and down arrows to review and enter to select. Touch device users, explore by touch or with swipe gestures.',
```

Comment:
Don't forget to integrate this with sitecore in the downstream apps (so that the default value can be overridden from the CMS).

Reply:
@vrojastpr It's hard for me to tell from just looking at the code, but make sure you're not doing this: https://adrianroselli.com/2019/10/stop-giving-control-hints-to-screen-readers.html